scripts/monitor-deployment.sh

```
@@ -81,17 +83,17 @@ while true; do
   
   # KV State
   echo "KV Configuration:"
-  ACTIVE_WORKER=$(wrangler kv:key get --binding=$MAINTENANCE_FLAGS_BINDING "active_worker" 2>/dev/null || echo "not set")
+  ACTIVE_WORKER=$("$WRANGLER" kv:key get --binding=$MAINTENANCE_FLAGS_BINDING "active_worker" 2>/dev/null || echo "not set")
   echo -e "  Active Worker: $(get_version_color $ACTIVE_WORKER)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)